code/__HELPERS/unsorted.dm

```diff
@@ -1743,3 +1743,14 @@ Game Mode config tags:
 
 	qdel(O)
 	return TRUE
+
+//Same as block(Start, End), but only returns the border turfs
+//'Start' must be lower-left, 'End' must be upper-right
+/proc/block_borders(turf/Start, turf/End)
+	ASSERT(istype(Start))
+	ASSERT(istype(End))
+
+	//i'm a lazy cunt and I don't feel like making this work
+	ASSERT(Start.x < End.x && Start.y < End.y)
```

Comment:
Need somebody to access if this is fine, or if I should fix it

(basically I made this proc runtime on purpose if the second turf isn't to the northeast of the first turf, as otherwise it gives weird results -- the runtime never happens if you use the proc with proper care and sanity)

Reply:
- [x] properly documented
- [x] properly runtimes if criteria are not met
- [x] accounting for improper parameters would require extra computations
I'd give this a pass